ansible/files/airflow/airflow.cfg

```diff
@@ -0,0 +1,195 @@
+[core]
+# The home folder for airflow, default is ~/airflow
+airflow_home = /usr/local/airflow
+
+# The folder where your airflow pipelines live, most likely a
+# subfolder in a code repository
+dags_folder = /usr/local/airflow/dags
+
+# The folder where airflow should store its log files. This location
+base_log_folder = /usr/local/airflow/logs
+# An S3 location can be provided for log backups
+# For S3, use the full URL to the base folder (starting with "s3://...")
+s3_log_folder = None
```

Comment:
This option has been deprecated, see the [latest example config](https://github.com/apache/incubator-airflow/blob/master/airflow/configuration.py#L174-L183)